### ISSUE TITLE
Add chokidar dependency to make this easier for Windows/Linux users

### ISSUE THREAD

**Issue by cdobbyn** [StephenGrider/ReduxSimpleStarter#115]
Not everyone is using MacOSX, could you consider include the following in devDependencies?
`"chokidar": "^1.6.1",`

This was pretty confusing as a new ReactJS user, as the tutorials were expecting the page to auto-reload,  the fsevents package only supports MacOSX due to using the related APIs. Once I added this, it was much more straightforward.

Thanks for your time.

**Comment by cdobbyn** [comment]
I suppose alternatively make a note about it in the tutorial, either would work.

**Comment by oskie-einhorn** [comment]
I have the same problem (Linux). Would you mind clarifying where exactly you inserted "chokidar": "^1.6.1"?  I'm also new to ReactJS and am a bit lost as to how to fix this problem. It would be a great help.

**Comment by cdobbyn** [comment]
Yeah no problem you can quickly install it into your project by running:
`npm install --save-dev chokidar`

This will download and install in addition to saving the dependency into the package.json file (in case you need to transfer between computers).

I did note that this package isn't perfect, I do occasionally need to restart the app to get it to re-compile my changes, but it's better than nothing.
